### ISSUE TITLE
Fix `wai-aria` error detected in `*.vue` component due to HTML comments

### ISSUE THREAD

**Issue by yamanoku** [markuplint/markuplint#1048]
- Markuplint version: `3.10.0`
- Parser lang: Vue.js
  - `@markuplint/vue-parser` version : `3.7.0`,
  - `@markuplint/vue-spec` version : `3.8.0`,
- Node.js version: `19.6.0`
- OS: macOS

## Describe the bug

Unintended error occurs when inserting HTML comment in `*.vue` component

## Code Example

```html
  <ul v-if="props.examples.length > 0">
    <!-- Error if comment inserted inside ul element -->
    <li v-for="item in props.examples" :key="item.id">
      {{ item.name }}
    </li>
  </ul>
```

Please see [example code](https://github.com/yamanoku-playground/2023-05-30-vue-template-comment-problem).

## Steps To Reproduce

1. git clone https://github.com/yamanoku-playground/2023-05-30-vue-template-comment-problem.git
1. npm ci
1. npm run markuplint

## The current behavior

```shell
<markuplint> error: Require the "listitem" role. Or, require aria-busy="true" (wai-aria) /2023-05-30-vue-template-comment-problem/src/ExampleComponentFailed.vue:13:3
  12: <template>
  13: ••<ul v-if="props.examples.length > 0">
  14: ••••<!--•Error•if•comment•inserted•inside•ul•element•-->
```

## The expected behavior

Error not detected.
